.styleguidist/styleguidist.sections.js

```diff
@@ -77,7 +78,7 @@ module.exports = {
               ]),
             },
             {
-              name: 'Icons',
+              name: 'Icon',
```

Comment:
Why this change? It has two separate components under the heading.

Reply:
It was to match it with the others. For example, Text and Link also contain several components, but they are in singular form in the navigation. This stood out.